source/frontend/src/features/mapSideBar/acquisition/add/AddAcquisitionForm.tsx

```diff
@@ -79,6 +73,62 @@ export const AddAcquisitionForm = React.forwardRef<
     }
   };
 
+  return (
+    <Formik<AcquisitionForm>
+      enableReinitialize
+      innerRef={ref}
+      initialValues={initialValues}
+      validationSchema={validationSchema}
+      onSubmit={handleSubmit}
+    >
+      {formikProps => {
+        return (
+          <>
+            <AddAcquisitionDetailSubForm
+              formikProps={formikProps}
+              onSubmit={onSubmit}
+              initialValues={initialValues}
+              showDiffMinistryRegionModal={showDiffMinistryRegionModal}
+              setShowDiffMinistryRegionModal={setShowDiffMinistryRegionModal}
+            ></AddAcquisitionDetailSubForm>
+          </>
```

Comment:
nit: no need to have the extra `<>` tag if we are returning only one component. please remove